.github/workflows/bcr/presubmit.yml

```diff
@@ -0,0 +1,10 @@
+bcr_test_module:
+    module_path: 'e2e/bzlmod'
+    matrix:
+        platform: ['debian10', 'macos', 'ubuntu2004', 'windows']
+    tasks:
+        run_tests:
+            name: 'Run test module'
+            platform: ${{ platform }}
+            test_targets:
+                - '//...'
```

Comment:
I couldn't find a spec for `presubmit.yml`. If we don't include build and test targets, does it automatically use `//...`?

Reply:
take a look in Yun's https://github.com/meteorcloudy/bzlmod-examples

Reply:
I took a look but he has the same example everywhere: https://github.com/meteorcloudy/bzlmod-examples/blob/main/registry/another/modules/A/1.1/presubmit.yml